modules/cnv-remove-boot-order-item-web.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assembly:
+//
+// * cnv/cnv_users_guide/cnv-edit-boot-order.adoc
+//
+
+[id="cnv-remove-boot-order-item-web_{context}"]
+
+= Removing items from a boot order list in the  web console
+
+Remove items from the boot order list.
+
+.Procedure
+
+. Click *Workloads* -> *Virtual Machines* from the side menu.
+. Select a virtual machine to open the *Virtual Machine Overview* screen.
+. Click the pencil icon net to *Boot Order* to add items to a boot order list.
```

Comment:
s/net/next (but specify which side it's on instead of just saying `next to`)

Reply:
Reworded: Click the pencil icon that is located on the right side of Boot Order.